libs/ui-toolkit/src/components/toggle/toggle.tsx

```diff
@@ -0,0 +1,57 @@
+import type { ChangeEvent } from 'react';
+
+// Supports controlled and uncontrolled setups.
+interface ToggleProps {
+  label: string;
+  value: string;
+}
+
+type TogglesArr = [
+  toggle1: ToggleProps,
+  toggle2: ToggleProps,
+  ...moreToggles: ToggleProps[]
+];
+
+export interface ToggleInputProps {
+  name: string;
+  toggles: TogglesArr;
+  className?: string;
+  onChange?: (e: ChangeEvent<HTMLInputElement>) => void;
+  checkedValue?: string;
+}
+
+export const Toggle = ({
+  name,
+  toggles,
+  className,
+  onChange,
+  checkedValue,
+  ...props
+}: ToggleInputProps) => {
+  return (
+    <fieldset className={`${className} flex`} {...props}>
```

Comment:
chore (non-blocking): can we use the classNames helper here?

Reply:
I wouldn't choose to myself here as I don't think it makes things any simpler, but if it creates consistency with other areas, sure